mozilla-services/consvc-shepherd
# [SHEP-136] - Fixing migration collison by manually rebuilding migrations

## References

[SHEP-123](https://mozilla-hub.atlassian.net/browse/SHEP-123)

## Problem Statement

[SHEP-125](https://github.com/mozilla-services/consvc-shepherd/pull/282) was merged into main (this has the 0023_alter_deliveredflight_campaign_name_and_more.py file in it) after [SHEP-77](https://github.com/mozilla-services/consvc-shepherd/pull/269/commits) already ran all it's tests and passed. In addition, there were no protections on main to prevent out of date branches from being merged in. So, SHEP-77 and SHEP-125 (both adding migration files) ended up with a naming collision.

## Proposed Changes

We've removed the migrations causing collisions and used:

	`python manage.py makemigrations`
	`python manage.py migrate`

To rebuild migrations with correct naming.

Additionally, @mashalifshin has fixed the github branch permissions so that we can no longer merge out of date branches to main from here on out. This should prevent issues like this from happening in the future!

## Verification Steps

Pull down the repo and verify that the migrations properly run, and the db docker containers builds successfully.

Credits also to: @mashalifshin @kauana @nanagb  @cyptm 

[SHEP-123]: https://mozilla-hub.atlassian.net/browse/SHEP-123?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[SHEP-125]: https://mozilla-hub.atlassian.net/browse/SHEP-125?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[SHEP-77]: https://mozilla-hub.atlassian.net/browse/SHEP-77?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
